request/request.d.ts

```diff
@@ -110,7 +110,7 @@ declare module 'request' {
 			key?: Buffer;
 			cert?: Buffer;
 			passphrase?: string;
-			ca?: Buffer;
+			ca?: string | Buffer | string[] | Buffer[];
```

Comment:
https://nodejs.org/dist/latest-v4.x/docs/api/https.html#https_https_request_options_callback

ca: A string, Buffer or array of strings or Buffers of trusted certificates in PEM format. If this is omitted several well known "root" CAs will be used, like VeriSign. These are used to authorize connections.